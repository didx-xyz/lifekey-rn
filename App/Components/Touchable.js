import React from 'react'

import {
  TouchableNativeFeedback,
  TouchableWithoutFeedback,
  Platform
} from 'react-native'

import Palette from '../Palette'

export default function(props) {
  if (Platform.OS == "android") {
    return (
      <TouchableNativeFeedback
        background={TouchableNativeFeedback.Ripple(Palette.consentGrayLight, true)}
        delayPressIn={0}
        {...props}
      >
        {props.children}
      </TouchableNativeFeedback>
    )
  }

  return (
    <TouchableWithoutFeedback
      delayPressIn={0}
      {...props}
    >
      {props.children}
    </TouchableWithoutFeedback>
  )
}
