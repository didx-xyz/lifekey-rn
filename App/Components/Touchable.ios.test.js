// external dependencies
import React from "react"

// internal dependencies
import Touchable from "./Touchable.ios"

test("Touchable renders correctly, by default", function() {
  const tree = shallow(
    <Touchable />
  )

  expect(tree).toMatchSnapshot()
})
