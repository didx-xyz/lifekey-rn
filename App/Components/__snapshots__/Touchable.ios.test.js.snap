// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Touchable renders correctly, by default 1`] = `
<TouchableWithoutFeedback
  delayPressIn={0}
/>
`;
